temporalcli/commands.activity_test.go

```diff
@@ -142,43 +142,34 @@ func (s *SharedServerSuite) TestActivityOptionsUpdate_Accept() {
 	s.ContainsOnSameLine(out, "MaximumAttempts", "5")
 }
 
-func (s *SharedServerSuite) TestActivityOptionsUpdate_Partial() {
```

Comment:
Did the partial update options test get removed?

Reply:
ah...
I was working on this while Update for on PR, and it looks like the merge did a bad job.
Yeah, I will add that test back, there is no reason to remove it.